tests/package.json

```diff
@@ -4,7 +4,7 @@
     "dev": "node server.js",
     "clean": "rm -rf .next static/manifest",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
+    "start": "yarn clean && yarn build && NODE_ENV=production node server.js",
```

Comment:
Hopefully this is okay, again, happy to revert this change, but it made the feedback loop considerably quicker whiel testing.

Reply:
We can keep this change, but it should be this if we go w/ it:

`yarn clean && NODE_ENV=production yarn build && NODE_ENV=production node`